stock_picking_product_interchangeable/models/stock_move.py

```diff
@@ -50,26 +45,64 @@ def _create_stock_move_interchangeable_products(self, products):
                     "location_dest_id": self.location_dest_id.id,
                     "company_id": self.company_id.id,
                 }
-                for product, qty in products
+                for product, qty in products_info
             ]
         )
 
+    @api.model
```

Comment:
why? self -> stock.move(), move -> stock.move(1)

you can use self intead of move variable

Reply:
Resolved